test/ranges/subset.js

```diff
@@ -15,6 +15,29 @@ const cases = [
   ['>2 <1', '3', true],
   ['1 || 2 || 3', '>=1.0.0', true],
 
+  // everything is a subset of *
```

Comment:
This comment should probably say something new, since this is not always true :sweat_smile: